packages/bullet-list/example/index.scss

```diff
@@ -1,4 +1,4 @@
-@import "~@fremtind/jkl-core/src/typography/root.scss";
+@import "~@fremtind/jkl-core/core.scss";
```

Comment:
import the entire core?

Reply:
Ah, its example, thats fine

Reply:
Yeah, I mostly just wanted to make sure it worked both ways!